models/general.js

```diff
@@ -0,0 +1,38 @@
+const constant = require("../constants");
```

Comment:
What's the difference between this file and /schemas/user.js?

Reply:
realized that there wasn't a difference as soon as i pushed, combined